components/rhs_comment/rhs_comment.jsx

```diff
@@ -282,6 +283,7 @@ class RhsComment extends React.PureComponent {
         const isSystemMessage = PostUtils.isSystemMessage(post);
         const isMeMessage = checkIsMeMessage(post);
         const fromAutoResponder = PostUtils.fromAutoResponder(post);
+        console.log(post);
```

Comment:
Looks like you left some debugging stuff by mistake. Otherwise, this looks good though